v2.3_release_notes.md

```diff
@@ -0,0 +1,88 @@
+Intel® Neural Compressor v2.3 Release
+---
+
+- Highlights
+- Features
+- Improvement
+- Productivity
+- Bug Fixes
+- Examples
+- Validated Configurations
+
+
+**Highlights**
+- Supported low precision (INT4, NF4, FP4) and [Weight-Only](https://github.com/intel/neural-compressor/blob/master/docs/source/quantization_weight_only.md) quantization algorithms including RTN, AWQ, GPTQ and TEQ on ONNX Runtime and PyTorch for LLMs optimization.
+- Supported quantization for ONNX Runtime DML EP and DNNL EP.
```

Comment:
ONNX Runtime DML EP on Intel NPU (e.g., Meteor Lake) and DNNL EP on Intel CPU (e.g., SPR)